src/app/pages/Swap/form/index.tsx

```diff
@@ -349,6 +355,16 @@ export const SwapForm = memo<Props>(({ account, slippageTolerance, onReview }) =
   const onSubmit = async () => {
     if (isSubmitting) return;
 
+    // TODO: add `toastUniqWarning(t('notEnoughTezForFee'), true)` call if the balance of the EVM native token is too low
```

Comment:
```suggestion
```
As far as I remember, this toast should appear after clicking on 'Max' button. Showing Wert modal is enough when you submit the form